src/PatchOrchestrationApplication/Common/WindowsUpdateOperationResult.cs

```diff
@@ -174,8 +185,8 @@ public static WindowsUpdateOperationResult Deserialize(string filePath)
         public override string ToString()
         {
             string windowsUpdateDetails = string.Join(",\n", UpdateDetails.Select(i => i.ToString()).ToArray());
-            return string.Format("WindowsUpdateOperationResult : OperationResult {0} , OperationTime {1}, OperationType {2}, UpdateDetails {3}, NodeName {4}, WindowsUpdateQuery {5}, WindowsUpdateFrequency {6}, RebootRequired {7}", 
-                                  OperationResult, OperationTime, OperationType, windowsUpdateDetails, NodeName, WindowsUpdateQuery, WindowsUpdateFrequency, RebootRequired);
+            return string.Format("WindowsUpdateOperationResult : OperationResult {0} , OperationTime {1}, OperationType {2}, UpdateDetails {3}, NodeName {4}, WindowsUpdateQuery {5}, WindowsUpdateFrequency {6}, RebootRequired {7}, OperationStartTime {8}", 
+                                  OperationResult, OperationTime, OperationType, windowsUpdateDetails, NodeName, WindowsUpdateQuery, WindowsUpdateFrequency, RebootRequired, OperationStartTime);
```

Comment:
>OperationStartTime [](start = 173, length = 18)

Log start and completion time together and change operation time to indicate operation completion time